internal/pkg/agent/cmd/install.go

```diff
@@ -21,7 +22,10 @@ import (
 	"github.com/elastic/elastic-agent/pkg/utils"
 )
 
-const flagInstallBasePath = "base-path"
+const (
+	flagInstallBasePath = "base-path"
+	flagInstallNonRoot  = "non-root"
```

Comment:
We want the default to be installation as root as that gives the best experience and allows the ability for Endpoint Security to be used. By changing to `--root` that would mean the default becomes non-root which is not what we want.

We will also not allow the ability user a specific user. It will always be the `elastic-agent` user and the `elastic-agent` group. They can always add a user to `elastic-agent` group to allow them to interact with the Elastic Agent.

Reply:
Agree, we can't change the default which means we need a negative in the flag like this.

My only other suggestion would be to come up with a name for this option that isn't Linux specific, since root doesn't exist on Windows. It's Administrator there. Perhaps `--unprivileged`? Not sure if that is as clear or if there is a better general term for this.

Or maybe just aliasing this to `--non-admin` works for people who only speak Windows.

Reply:
I like `--unprivileged`, it works for both. Instead of having multiple flags depending on the OS. If there is an agreement on `--unprivileged` I will change it to that.

Reply:
i also agree with `--unprivileged` we need to ensure this doesn't cause a breaking change.

Reply:
Changed to `--unprivileged`.